Makefile

```diff
@@ -95,6 +95,14 @@ E2E_TEST_FLAGS = "--skip-deletion=false" -timeout 25m # See README.md, default g
 # see target "image-build"
 IMAGE_BUILD_FLAGS ?= --build-arg USE_LOCAL=false
 
+# Prometheus-Unit Tests Parameters
+PROMETHEUS_CONFIG_YAML ?= $(shell pwd)/config/monitoring/prometheus/apps/prometheus-configs.yaml
+PROMETHEUS_CONFIG_DIR ?= $(shell pwd)/config/monitoring/prometheus/apps/
+GENERATED_ALERT_DIR ?= $(shell pwd)/tests/prometheus_unit_tests/
+PROMETHEUS_IMAGE ?= quay.io/prometheus/prometheus
+CONTAINER_RUNTIME ?= docker
+CRITICAL_SEVERITY="critical"
```

Comment:
Unused

Reply:
Removed the above, for the `CRITICAL_SEVERITY` it needs to be used. Updating this.

Reply:
Updated this as well.